jaxopt/_src/scipy_wrappers.py

```diff
@@ -211,19 +211,16 @@ class ScipyWrapper(abc.ABC):
       methods relying on FORTRAN code, such as the `L-BFGS-B` solver for
       `scipy.optimize.minimize`, require casting to float64.
     jit: whether to JIT-compile JAX-based values and grad evals.
-    implicit_diff: if True, enable implicit differentiation using cg,
-      if Callable, do implicit differentiation using callable as linear solver.
-      Autodiff through the solver implementation (`implicit_diff = False`) not
-      supported. Setting `implicit_diff` to False will thus make the solver
-      not support JAX's autodiff transforms.
+    implicit_diff_solve: the linear system solver to use.
     has_aux: whether function `fun` outputs one (False) or more values (True).
       When True it will be assumed by default that `fun(...)[0]` is the
       objective.
   """
   method: Optional[str] = None
   dtype: Optional[Any] = onp.float64
   jit: bool = True
-  implicit_diff: Union[bool, Callable] = False
+  implicit_diff: bool = False
```

Comment:
Appears to be missing from docstring. Besides, since autodiff via unrolling isn't supported by the SciPy wrappers, should we just remove the attribute?

Reply:
When `run` is decorated with `@custom_root`, which happens if `implicit_diff=True`, keyword arguments are no longer usable in `run`. I wanted to remove `implicit_diff` here but this broke some of your tests using keyword arguments. So I'm leaving `implicit_diff` temporarily until we fix the keyword issue (@froystig and I think we have a fix).

Reply:
Makes sense. Please let me know if you'd prefer me to change the tests to not rely on kwargs.

Reply:
No that's fine. @froystig and I will tackle the issue soon.